Examples/OneSignalDemo/app/src/main/assets/api_key.txt

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJSU0EtU0hBMjU2IiwidmVyIjoiMSJ9.eyJ2ZXIiOiIzIiwiZW5kcG9pbnRzIjp7ImF1dGh6IjoiaHR0cHM6Ly93d3cuYW1hem9uLmNvbS9hcC9vYSIsInRva2VuRXhjaGFuZ2UiOiJodHRwczovL2FwaS5hbWF6b24uY29tL2F1dGgvbzIvdG9rZW4ifSwiY2xpZW50SWQiOiJhbXpuMS5hcHBsaWNhdGlvbi1vYTItY2xpZW50LjE4YzMyNDljZGU0YzQ3YTI5YmZmZDQ5NTFiZTVmMjRlIiwiYXBwRmFtaWx5SWQiOiJhbXpuMS5hcHBsaWNhdGlvbi43MDMxNjFhNTk5NjM0MWY0ODI2N2RmNWU2Yzc1MTM5MiIsImlzcyI6IkFtYXpvbiIsInR5cGUiOiJBUElLZXkiLCJwa2ciOiJjb20ub25lc2lnbmFsLnNka3Rlc3QiLCJhcHBWYXJpYW50SWQiOiJhbXpuMS5hcHBsaWNhdGlvbi1jbGllbnQuZGUxZDljYTk3MDk3NGRmYjlmMDVkMjg3YWEzNDhjYjEiLCJ0cnVzdFBvb2wiOm51bGwsImFwcHNpZ1NoYTI1NiI6IkQyOjgzOjBDOjAyOkU2OkExOjYzOkZEOkU1OkYxOjdDOjM4OkI2Ojk1OkM3OkVBOjBCOjQyOkQ1OkY5OjlDOjMyOkRGOkI1OjUzOkFEOjU0OjJEOjc2OkNFOkY5OkE3IiwiYXBwc2lnIjoiRTU6OTY6RkY6RTU6RjU6RDg6QkM6MDk6Qzk6QUY6OTA6RkU6Rjk6QUY6RUY6QTYiLCJhcHBJZCI6ImFtem4xLmFwcGxpY2F0aW9uLWNsaWVudC5kZTFkOWNhOTcwOTc0ZGZiOWYwNWQyODdhYTM0OGNiMSIsImlkIjoiOTA4ZDZhMzgtOGFhMy00Zjg4LTg3NWUtNzM3YjdlMzNkMzNmIiwiaWF0IjoiMTYwMjU1NTYyMTgzMSJ9.EhgKSIl9JKbVIe+CNeN0WkFbGoMJpAZEX0jq3mXTGv2kX+DN/GdFpJkIlOQ30dF1o7CLgWz4lxK0AHUAmeuNdGzyBAf7EmzqRukGmQ6Gh5sgo+YoecQnRFXSMv/uRt9KIDYWY9YvOWeNxfrWPOu6JNfLRgrFt/gmtDEk/THIOS5UDtRJW59+x3e6aVOXNJBt7Vo8gaUgqE7JK/p/iGRyT5AnaTDQICrlIomYFg6ZrJ61zQ0ATg+e5GTGeURcOMWDff8lMdfOC3pDGUGCW574aslTlV1/dCIbWLJ6bbxvcZNwAB/eU7F3XvwANDY4xvQm4V9RJz37zcWJ2rObz2P0/w==
```

Comment:
is this something necessary? do we want this to be pushed?

Reply:
It is going to be different based on the package name and the Android Signing keys used to build your app.

I think it would be good to have this file here but instead the contents of this file should point to our docs on how they can generate this file.